bin/logstash.lib.sh

```diff
@@ -100,7 +100,8 @@ setup_java() {
       if [ -x "$LS_JAVA_HOME/bin/java" ]; then
         JAVACMD="$LS_JAVA_HOME/bin/java"
         if [ -d "${LOGSTASH_HOME}/${BUNDLED_JDK_PART}" -a -x "${LOGSTASH_HOME}/${BUNDLED_JDK_PART}/bin/java" ]; then
-          echo "WARNING: Using LS_JAVA_HOME while Logstash distribution comes with a bundled JDK."
+          BUNDLED_JDK_VERSION=`cat JDK_VERSION`
+          echo "WARNING: Using LS_JAVA_HOME while Logstash distribution comes with a bundled JDK(${BUNDLED_JDK_VERSION})."
```

Comment:
In mac, I got these message
```
Using LS_JAVA_HOME defined java: /Users/kcheng/.sdkman/candidates/java/current.
WARNING: Using LS_JAVA_HOME while Logstash distribution comes with a bundled JDK(Temurin-11.0.14.1+1).
OpenJDK 64-Bit Server VM warning: Option UseConcMarkSweepGC was deprecated in version 9.0 and will likely be removed in a future release.
```

I suggest giving a direction to the user what need to be done to resolve the warning

```
WARNING: Logstash distribution comes with a bundled JDK(Temurin-11.0.14.1+1), which is a known-working and tested java distribution. You are recommended to use the bundled JDK.
```

Reply:
Thanks' for suggestion, that message was already present for some releases.
To obtain that message a user has to intentionally set `LS_JAVA_HOME` so the resolution should be immediate.